src/main/webapp/app/overview/submission-result-status.component.ts

```diff
@@ -48,7 +49,23 @@ export class SubmissionResultStatusComponent implements OnChanges {
             this.quizNotStarted = ArtemisQuizService.notStarted(quizExercise);
         } else {
             this.uninitialized = !afterDueDate && !this.studentParticipation;
-            this.notSubmitted = !!this.studentParticipation && !this.studentParticipation.submissions?.length;
+            this.notSubmitted = afterDueDate && !!this.studentParticipation && !this.studentParticipation.submissions?.length;
```

Comment:
Is `notSubmitted` is only after the deadline? What about the case when student haven't submitted _yet_?

Reply:
This is explained in the PR description. Would you like a comment here as well?

Reply:
Makes sense